ui/components/ui/page-container/page-container-header/page-container-header.component.js

```diff
@@ -49,6 +49,7 @@ export default class PageContainerHeader extends Component {
       onClose && (
         <div
           className="page-container__header-close"
+          data-testid="header-close-button"
           onClick={() => onClose()}
```

Comment:
Above looks wrong it should rather be a button, may be a role can be added for unit test.

Reply:
fixed here: https://github.com/MetaMask/metamask-extension/pull/13271/commits/2c635b32bc267b68a43bd0683f4be4b997fa1d58